c++/test/TestReader.cc

```diff
@@ -105,6 +105,15 @@ namespace orc {
                                                         nextSkippedRows));
   }
 
+  void prefetchAllStripes(Reader* reader) {
+    auto num_stripes = reader->getNumberOfStripes();
+    std::vector<int> stripes;
+    for (size_t i = 0; i < num_stripes; ++i) {
+      stripes.push_back(i);
+    }
+    reader->preBuffer(stripes, {0});
```

Comment:
Please add test case where preBuffer is called for multiple times and with different stripe/column, etc.